### PR TITLE
fetch default dhcp interface for 6.1

### PR DESCRIPTION
**Pull request by sghai**
handles #13

### CODE REVIEW

**Comment by jyejare** [comment]
ACK, This should also remove DHCP_INTERFACE field from Satellite Upgrade job.

File: roles/sat6repro/tasks/main.yml [hunk 1]
```diff
@@ -78,8 +78,12 @@
 - name: untar config files
   command: tar --selinux --overwrite -xvf /tmp/config_files.tar.gz  -C /
   when: perform_restore
+- name: Identify the default dhcp interface
+  shell: route | grep -m 1 '^default' | awk '{print $8}'
```
Comment (pgagne) [inline roles/sat6repro/tasks/main.yml]: I don't know if we want to assume that the default route interface is the one used for DHCP. This seems dangerous.

**Comment by sthirugn** [comment]
https://github.com/RedHatSatellite/satellite-clone/issues/13#issuecomment-274080330

**Comment by sthirugn** [comment]
@sghai please see https://github.com/RedHatSatellite/satellite-clone/issues/13#issuecomment-274955627

**Comment by sghai** [comment]
thanks @sthirugn for updates.

**Pull request closed without merging**